core-java/src/main/java/com/baeldung/passwordhashing/SHA512Hasher.java

```diff
@@ -0,0 +1,35 @@
+package com.baeldung.passwordhashing;
+
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+
+/** A really simple SHA_512 Encryption example.
+ *
+ */
+public class SHA512Hasher {
+
+  public String get_SHA_512_SecurePassword(String passwordToHash, byte[] salt){
```

Comment:
Could we name and design this the same as the equivalent in PBKDF2Hasher? I think the symmetry will help readers find commonalities.